fragments/story.txt

```diff
@@ -83,3 +82,11 @@ Sure, Godus is a mess, but what ever happened to Peter Molyneux's <game>?
 PS Plus users find themselves elated to discover both <game_a> and <game_b> will be free this month.
 After an internal feud concerning the development of <game>, <luminary>'s name is being removed from public material.
 The wait for new <platform> games continues.
+Puzzling. Confusing. Terrifying. How did <game> make it through Steam Greenlight?
+The news of <company>'s closure has been met with uproar, after being announced on-stage at <event>.
+Rumours are circulating about Netflix creating a live-action series of <comapny>'s <game>.
+Rhythm games are coming back to consoles, but will we see support for the <peripheral>?
```

Comment:
Yes, 'the Amiibo' is nonsense. This is why there's no 'the' in the version I proposed.

If you want to add changes to this PR, you'll probably want to edit at <https://github.com/euricaeris/rubbersuits/blob/patch-4/fragments/story.txt>.